src/OMSimulatorLib/OMSimulator.cpp

```diff
@@ -89,7 +89,7 @@ oms_status_enu_t oms3_rename(const char* cref_, const char* newCref_)
   if (cref.isValidIdent())
     return oms3::Scope::GetInstance().renameModel(cref, newCref);
   else
-    return logError("Only implemented for model identifiers");
+    return logError("oms3_rename failed. Only implemented for model identifiers");
```

Comment:
When using `logError` the current function name is added as a prefix to the error message. I introduced this change with the first commit of this PR.

Reply:
Are you sure? I don't get any function name prefixed with the message.